app/plugin_api.go

```diff
@@ -171,6 +171,13 @@ func (api *PluginAPI) DeleteUser(userId string) *model.AppError {
 	return err
 }
 
+func (api *PluginAPI) GetUsers(page int, perPage int) ([]*model.User, *model.AppError) {
```

Comment:
I'm thinking we ought to accept a `model.UserGetOptions` here to expose the full flexibility of the API. Otherwise, we'll constantly need variants for the myriad of ways users might be fetched.

Does that seem reasonable, @dom3k?

Reply:
Same thing for: #10096

Reply:
@lieut-data What do you think about dropping `page` and `perPage` and using `GetUsers(options model.UserGetOptions) ([]*model.User, *model.AppError)` as `model.UserGetOptions` contains `Page` and `PerPage`?

cc @jwilander as you created the ticket

Reply:
OK, I changed it.

Reply:
Good with me 👍